.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+- React-Native tools version:
```

Comment:
Actually this line is meant to be our extension.
I don't think we usually care about the react-native-cli version, it usually just passes stuff through to the react-native library.